[[ Installer ]] Fix behavior resolution in installer

montegoulding opened the pull request:
This patch fixs behavior resolution in the installer that was broken due
to a previous patch changing the timing of behavior resolution for the
startup stack.

runrevmark commented:
@livecode-vulcan review ok 47a58b1

livecode-vulcan commented:
💙 review by @runrevmark ok 47a58b1ad48c14a5cef09d6aca5ad7e725a16dba

livecode-vulcan commented:
😎 test success 47a58b1ad48c14a5cef09d6aca5ad7e725a16dba

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1100)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1116)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1090)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/43)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/43)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1095)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1117)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1100)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1116)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1099)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1109)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1132)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1095)

montegoulding committed:
[[ Installer ]] Fix behavior resolution in installer

This patch fixs behavior resolution in the installer that was broken due
to a previous patch changing the timing of behavior resolution for the
startup stack.

montegoulding commented:
this is just a force push of the same patch to make travis happy so no need to re-review
